warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -387,6 +393,7 @@ protected DefaultQueryPlanner(DefaultQueryPlanner other) {
         rangeStreamClass = other.rangeStreamClass;
         setSourceLimit(other.sourceLimit);
         setPushdownThreshold(other.getPushdownThreshold());
+        setRegexRewriteOptions(other.getRegexRewriteOptions());
```

Comment:
Why not simply create more transform rules (see 2 lines below) instead of creating a whole new injected set of rewrite options?

Reply:
These are not simple transform rules. These rules are more tailored and intended to be run before and after index expansion.

Reply:
but.... as far as I can tell you are simply transforming nodes within the tree.  I think I need to sit down with you to understand why a separate visitor.  The node transform rules are supplied the node and hence its subtree, so they can do whatever they need to keep the query executable.